frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx

```diff
@@ -13,38 +13,27 @@ interface realDoneProps {
   label: string
 }
 
-function getSelectedDoneColumns(selectedBoardColumns: { name: string; value: string }[]) {
-  const selectedDoneColumns = selectedBoardColumns
-    .filter(({ value }) => value === METRICS_CONSTANTS.doneValue)
-    .map(({ name }) => name)
-  return selectedDoneColumns
-}
+const getSelectedDoneColumns = (selectedBoardColumns: { name: string; value: string }[]) =>
+  selectedBoardColumns.filter(({ value }) => value === METRICS_CONSTANTS.doneValue).map(({ name }) => name)
 
-function getFilteredStatuses(
+const getFilteredStatuses = (
   columns: { key: string; value: { name: string; statuses: string[] } }[],
   selectedDoneColumns: string[]
-) {
-  const filteredStatuses = columns
-    .filter(({ value }) => selectedDoneColumns.includes(value.name))
-    .flatMap(({ value }) => value.statuses)
-  return filteredStatuses
-}
+) => columns.filter(({ value }) => selectedDoneColumns.includes(value.name)).flatMap(({ value }) => value.statuses)
 
-function getDoneStatuses(columns: { key: string; value: { name: string; statuses: string[] } }[]) {
-  const doneStatuses =
-    columns.find((column) => column.key === METRICS_CONSTANTS.doneKeyFromBackend)?.value.statuses ?? []
-  return doneStatuses
-}
+const getDoneStatuses = (columns: { key: string; value: { name: string; statuses: string[] } }[]) =>
```

Comment:
Statuses?

Reply:
Sorry, it should be Status, I'll fix it right now